ts/components/FiscalCodeLandscapeModal.tsx

```diff
@@ -0,0 +1,98 @@
+import { Body, Button, Container, Content, View } from "native-base";
+import * as React from "react";
+import { Modal, Platform, StyleSheet } from "react-native";
+import { isIphoneX } from "react-native-iphone-x-helper";
+import { UserProfile } from "../../definitions/backend/UserProfile";
+import IconFont from "../components/ui/IconFont";
+import customVariables from "../theme/variables";
+import FiscalCodeComponent from "./FiscalCodeComponent";
+import AppHeader from "./ui/AppHeader";
+
+type Props = Readonly<{
+  profile: UserProfile;
+  isVisible: boolean;
+  onClose: () => void;
+  showBackSide: boolean;
+}>;
+
+const globalHeaderHeight: number =
+  Platform.OS === "ios"
+    ? isIphoneX()
+      ? customVariables.appHeaderHeight + 42
+      : customVariables.appHeaderHeight + 18
+    : customVariables.appHeaderHeight;
+
+const styles = StyleSheet.create({
+  content: {
+    backgroundColor: customVariables.brandDarkGray,
+    paddingHorizontal: customVariables.contentPadding,
+    marginTop: -customVariables.appHeaderHeight,
+    paddingTop: 0
+  },
+
+  headerSpacer: {
+    height: customVariables.appHeaderHeight
+  },
+
+  closeButton: {
+    position: "absolute",
+    right: customVariables.contentPadding,
+    top: globalHeaderHeight - customVariables.appHeaderHeight
+  }
+});
+
+export default class FiscalCodeLandscapeModal extends React.PureComponent<
+  Props
+> {
+  private contentRef: any;
+
+  public render() {
+    return (
+      <Modal
```

Comment:
The only react-native Modal must be the IdentificationModal for all the other "modals" use the LightModal component (withLightModal HOC).

Reply:
can I propose you to open a new story that includes the migration of components that now use Modal to the LightModal (AbortOnboardingModal, ContextualHelp, FiscalCodeLandscapeModal, ConfirmCardDetailsScreen)? cc @fpersico